src/style.css

```diff
@@ -0,0 +1,22 @@
+
+body {
+  font-family: Arial;
+  font-size: 13px;
+  line-height: normal;
+}
+
+.h1-heading {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
+}
+
+.h2-heading {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+a {
```

Comment:
What would be if we have a few `a` in the document and we want to style it in another way?

Reply:
ok,thanks